sortandsearch.c

```diff
@@ -0,0 +1,128 @@
+#include <stdio.h>
+//#include <cs50.h>
+#include <stdbool.h>
+
+void sort(int values[], int n);
+bool search(int value, int values[], int n);
+
+int main(void) {
+    int values[4];
+    values[0] = 39;
+    values[1] = 40;
+    values[2] = 41;
+    values[3] = 42;
+    //printf("values before sorting: \n");
+    //int y;
+    //for (y = 0; y < 5; y++) {
+    //    printf("values: %d\n", values[y]);
+    //}
+    
+    sort(values, 4);
+    //printf("values after sorting: \n");
+    //for (y = 0; y < 5; y++) {
+    //    printf("values: %d\n", values[y]);
+    //}
+    
+    bool found = search(42, values, 4);
+    if (found == true) {
+        printf("Found the value\n");
+    }
+    else {
+        printf("Didn't find it: %d\n", found);
+    }
+}
+
+
+
+void sort(int values[], int n)
```

Comment:
If you make some helper functions, this function will be much shorter and easier to read.

```
find_smaller_between(int values[], int value, int start, int end);
swap(int values[], int i, int j);
etc..
```